python/createModels/createModels.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+from sklearn.linear_model import LinearRegression
```

Comment:
Should test more models! Take a look at my code (it's not good though, but it displays a little you might learn something from)